cclib/bridge/__init__.py

```diff
@@ -21,5 +21,13 @@
 if find_package("psi4"):
     from cclib.bridge.cclib2psi4 import makepsi4
 
-
+if find_package("horton"):
+    try:
+        from horton import __version__
+    except Exception:
+        pass
+    if (__version__[0] == '2'):
+        from cclib.bridge.cclib2horton import makehorton
+    # TODO: Add something for iodata (Horton 3) later
```

Comment:
Could you reference an issue here (maybe create one)?

Reply:
@langner I am planning to add bridge for horton 3 today or tomorrow. Thanks!

Reply:
This would be a good thing to send in a separate PR.

Reply:
Yes -- I will try to split this into a separate PR tomorrow. The commits for horton 2 and 3 are readily separable, so I think should be able to do so without much issue. Thanks.